app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,11 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      if(exportData.Keystore.length > 0) {
+        alert("The key will be exported and should be protected");
```

Comment:
i wonder what could be the easiest way to manage locations within this node js framework if we in the future translated openHIM into other languages!